supportedBackends/local/src/test/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+local-config {
+  root = "local-cromwell-executions"
+  filesystems = {
+    local {
+      localization: [
+        "hard-link", "soft-link", "copy"
+      ]
+    }
+  }
+}
```

Comment:
Yeah I thought I would use this one as a default when I don't really care about the config, and then override it with a custom one when I do.